lib/domain/usecase/send_image_interactor.dart

```diff
@@ -17,7 +16,7 @@ class SendImageInteractor {
     final matrixFile = await entity.toMatrixFile();
     if (matrixFile != null) {
       try {
-        final mxcUri = await room.sendImageFileEvent(
+        await room.sendImageFileEvent(
```

Comment:
what?

Reply:
Because we don't use it. I removed it and made it clean